docs/glossary.md

```diff
@@ -50,6 +50,35 @@ In each block, five tickets are called to vote. In addition to votes on any open
 
 ---
 
+#### Change
+
+Decred has a concept of change, which is very similar to the concept of change
+when using physical cash.
+
+To explain by way of an example, imagine you have received 5 DCR in a single
```

Comment:
While this is a useful example, it doesn't fit well with the general form of the other definitions, which are more concise and don't have examples fleshed out. Also, I think this general example/explanation will available elsewhere on the net if someone is curious to learn more.